go/logic/migrator.go

```diff
@@ -219,7 +219,11 @@ func (this *Migrator) onChangelogStateEvent(dmlEvent *binlog.BinlogDMLEvent) (er
 // listenOnPanicAbort aborts on abort request
 func (this *Migrator) listenOnPanicAbort() {
 	err := <-this.migrationContext.PanicAbort
+
+	//should cleanup before exit.
+	this.finalCleanup()
```

Comment:
there's not much to clean up, really, just the remains of the `_gho` table. We don't have often critical errors, or at least we work hard not to have often critical errors.